apps/E2E/src/Checkbox/specs/Checkbox.spec.windows.ts

```diff
@@ -1,8 +1,7 @@
 import NavigateAppPage from '../../common/NavigateAppPage';
 import CheckboxPageObject from '../pages/CheckboxPageObject';
```

Comment:
Why do we have two test specs for ".win.ts" and ".windows.ts"? Can win32 and windows not use the same test spec?

Reply:
The .windows.ts spec states that an appium method for checking whether the checkbox is toggled does not work on the UWP app. Therefore, functional tests in the .win.ts spec will always fail.

Reply:
Gotcha, thanks for clarifying!